hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/Tables/details/ReviewOrgUnitChangesDetailsTableBody.tsx

```diff
@@ -24,17 +26,38 @@ export const ReviewOrgUnitChangesDetailsTableBody: FunctionComponent<Props> = ({
 }) => {
     return (
         <TableBody>
-            {newFields.map(field => (
-                <ReviewOrgUnitChangesDetailsTableRow
-                    key={field.key}
-                    field={field}
-                    setSelected={setSelected}
-                    isNew={isNew}
-                    isNewOrgUnit={isNewOrgUnit}
-                    changeRequest={changeRequest}
-                    isFetchingChangeRequest={isFetchingChangeRequest}
-                />
-            ))}
+            {newFields.map(field => {
+                if (field.key === 'groups') {
+                    const changedFieldWithNewValues =
+                        changeRequest && changeRequest[`new_${field.key}`];
+                    const changedFieldWithOldValues =
+                        changeRequest && changeRequest[`old_${field.key}`];
+
+                    return (
+                        <HighlightFields
+                            label={field.label}
```

Comment:
As mentioned above, this logic is applicable to org unit types too. May be consider a way to differentiate fields type in usenewFields and add a type 'array' or 'string' that can help you to show your component or not